src/hyp3_isce2/__main__.py

```diff
@@ -21,12 +22,17 @@ def main():
         default='insar_tops_burst',
         help='Select the HyP3 entrypoint to use',  # HyP3 entrypoints are specified in `pyproject.toml`
     )
+    parser.add_argument('++omp-num-threads', type=int, help='The number of OpenMP threads to use for parallel regions')
```

Comment:
What happens if the user doesn't specify this? Do we need a default? If so, I'd suggest 8.

Reply:
No idea, I just copied the `hyp3-gamma` implementation. I'll merge for now and we can circle back to it before merging to `develop`, if we feel it's important.

Reply:
Not setting it lets ISCE2/OpenMP use *their* defaults, which I believe is the behavior we'd want.